apps/skilavottord/web/screens/AccessControl/AccessControl.tsx

```diff
@@ -154,15 +148,15 @@ const AccessControl: FC = () => {
 
   const accessControls = accessControlsData?.skilavottordAccessControls || []
 
-  const partners = recyclingPartnerData?.skilavottordAllRecyclingPartners || []
-  const recyclingPartners = filterInternalPartners(partners).map((partner) => ({
-    label: partner.companyName,
-    value: partner.companyId,
-  }))
+  // const recyclingPartners = filterInternalPartners(partners).map((partner) => ({
+  //   label: partner.companyName,
```

Comment:
Let's finish this work @miamiultras .

Reply:
done, but I had to revert that change and make recyclingPartner query anyway, this is needed for creating access control so user can pick from all possible recycling companies